sql.txt

```diff
@@ -0,0 +1,96 @@
+CREATE DATABASE auction
+    WITH 
+    OWNER = postgres
+    ENCODING = 'UTF8'
+    LC_COLLATE = 'English_United States.1252'
+    LC_CTYPE = 'English_United States.1252'
+    TABLESPACE = pg_default
+    CONNECTION LIMIT = -1;
+
+CREATE TABLE public.items
+(
+    id integer NOT NULL,
+    name text COLLATE pg_catalog."default",
+    "desc" text COLLATE pg_catalog."default",
+    img text COLLATE pg_catalog."default",
+    price integer,
+    category text COLLATE pg_catalog."default",
+    CONSTRAINT items_pkey PRIMARY KEY (id)
+)
+
+
+TABLESPACE pg_default;
+
+ALTER TABLE public.items
+    OWNER to postgres;
+
+CREATE TABLE public.users
+(
+    id integer NOT NULL,
+    name text COLLATE pg_catalog."default",
+    password text COLLATE pg_catalog."default",
+    email text COLLATE pg_catalog."default",
+    img text COLLATE pg_catalog."default",
+    CONSTRAINT users_pkey PRIMARY KEY (id)
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE public.users
+    OWNER to postgres;
+
+CREATE TABLE public.bids
+(
+    id integer NOT NULL,
+    item_id integer,
+    user_id integer,
+    bid_price integer,
+    "time" time without time zone,
+    date date,
+    CONSTRAINT bids_pkey PRIMARY KEY (id)
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE public.bids
+    OWNER to postgres;
+
+
+
+
+INSERT INTO public.items(
```

Comment:
Please do not insert anything in db creation script. Db creation script should only create db tables/constraints/rules, not insert dummy data. 
If you want to have script to seed data, have another one.